script/start_server.sh.sample

```diff
@@ -50,6 +50,19 @@ if ! command -v $PUMA_CMD 2>&1 >/dev/null; then
   exit 1
 fi
 
+# Pick a system stack:
+# UNCOMMENT AS REQUIRED ON NON SAMPLE FILE
+# STACK="beta"
+# STACK="live"
+
```

Comment:
It would be cool if `STACK` could be derived if possible, when not given (with the prompt as a fallback):

```suggestion
# If not given, attempt to derive stack from application user:
if [ -z "$STACK" ]; then
  if [[ $USER =~ _live$ ]]; then
    STACK="live"
  elif [[ $USER =~ _beta$ ]]; then
    STACK="beta"
  fi
fi
```

Reply:
oops. Yes in my mind I thought was doing biobank which I thought had no username difference